ggml/include/ggml.h

```diff
@@ -559,10 +559,10 @@ extern "C" {
 
     enum ggml_log_level {
         GGML_LOG_LEVEL_NONE  = 0,
-        GGML_LOG_LEVEL_INFO  = 1,
-        GGML_LOG_LEVEL_WARN  = 2,
-        GGML_LOG_LEVEL_ERROR = 3,
-        GGML_LOG_LEVEL_DEBUG = 4,
+        GGML_LOG_LEVEL_DEBUG = 1,
+        GGML_LOG_LEVEL_INFO  = 2,
+        GGML_LOG_LEVEL_WARN  = 3,
+        GGML_LOG_LEVEL_ERROR = 4,
         GGML_LOG_LEVEL_CONT  = 5, // continue previous log
     };
```

Comment:
I also reordered this because I think it is important that the levels are ordered by severity so that they can be filtered. Unfortunately `GGML_LOG_LEVEL_CONT` kind of ruins that.

Reply:
Hmm I think `GGML_LOG_LEVEL_CONT` could be something equivalent to `INFO` or `DEBUG`?